<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 2</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Перенос</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="19">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>731</width>
        <height>444</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Сдвиг по x:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditX">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Сдвиг по y:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditY">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="moveButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Переместить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Поворот</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>x центра:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditX_2">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>y центра:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditY_2">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Угол:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditAngle">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="rotateButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Повернуть</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Масштабирование</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>x центра:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEditX_3">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>y центра:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEditY_3">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>dx:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEditDX">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>dy:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEditDY">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="scaleImageButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Изменить масштаб</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QPushButton" name="discardButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QPushButton" name="discardAllButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>145</width>
        <height>149</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QPushButton" name="aboutButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>О программе</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="2">
     <widget class="QPushButton" name="authorButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Об авторе</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="2">
     <widget class="QPushButton" name="outButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
